src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -97,6 +98,8 @@ IOktaApi oktaApi
 	private void CacheOktaSession( string userId, string org, string sessionId, DateTimeOffset expiresAt ) {
 		var session = new OktaSessionCache( userId, org, sessionId, expiresAt );
 		var existingSessions = ReadOktaSessionCacheFile();
+		existingSessions = existingSessions.Where( session => session.UserId != userId && session.Org != org )
```

Comment:
naming nit (it's not "existing sessions" anymore):
```suggestion
		var sessionsToCache = existingSessions.Where( session => session.UserId != userId && session.Org != org )
```
also at [L105](https://github.com/Brightspace/bmx/pull/359/commits/67b27fb22b31d3104ac9d7bbdfa59bb4d3aec04e#diff-640013b3685b8a2cbcee8c5b2e26b1afc631ea22dc293c44a6f402feab21ff73R105)